client/spec/search_spec.js

```diff
@@ -71,4 +73,40 @@ describe('Search', function() {
         priority3.click();
         expect(element.all(by.repeater('items._items')).count()).toBe(1);
     });
+
+    it('can search by from desk field', function() {
+        workspace.switchToDesk('SPORTS DESK').then(content.setListView);
+        expect(element.all(by.repeater('items._items')).count()).toBe(2);
```

Comment:
element.all(by.repeater('items._items')) should be moved in a search helper method